indico/modules/events/clone.py

```diff
@@ -174,10 +174,7 @@ def _clone_contrib_settings(self, new_event):
         })
 
     def _clone_subcontrib_settings(self, new_event):
-        subcontribution_settings_data = subcontribution_settings.get_all(self.old_event)
-        subcontribution_settings.set_multi(new_event, {
-            'speakers_can_submit': subcontribution_settings_data['speakers_can_submit'],
-        })
+        new_event.speakers_can_submit = self.old_event.speakers_can_submit
```

Comment:
I'd move this into `_clone_protection` since the session is related to the stuff in there